pkg/workflows/management/install_new_components.go

```diff
@@ -0,0 +1,41 @@
+package management
+
+import (
+	"context"
+
+	"github.com/aws/eks-anywhere/pkg/task"
+	"github.com/aws/eks-anywhere/pkg/workflows"
+)
+
+type installNewComponents struct{}
+
+// Run installNewComponents performs actions needed to upgrade the management cluster.
+func (s *installNewComponents) Run(ctx context.Context, commandContext *task.CommandContext) task.Task {
+	if commandContext.UpgradeChangeDiff.Changed() {
```

Comment:
I think we should do the apply no matter what
Even if `UpgradeChangeDiff` doesn't have any diff, it's possible that we have a new bundle, hence the controller will need it.

Reply:
So I'm proposing removing this `if`